src/modules/ekf2/ekf2_main.cpp

```diff
@@ -489,9 +489,9 @@ void Ekf2::task_main()
 		}
 
 		// read airspeed data if available
-		float eas2tas = airspeed.true_airspeed_m_s / airspeed.indicated_airspeed_m_s;
-
 		if (airspeed_updated && airspeed.true_airspeed_m_s > 7.0f) {
```

Comment:
I think the 7 m/s barrier is way too low. It should be around 4 m/s because a small plane will fly just fine around 8 m/s.